npm-update

```diff
@@ -25,7 +25,11 @@
 # Dependencies where minor updates break, only bump microversion
 FRAGILE = [
     "jquery",
-    "patternfly",
```

Comment:
This is PF3, `@patternfly` is PF4. This means, that PF3 is now not fragile an PF4 is updated as a group.

Reply:
Right, most projects don't use pf3 any more (all but cockpit itself). If we leave this, it will cause PF4 to only get updated to microversions, which is not what we want IMHO. Indeed this will now attempt to update PF3 to new minorversions also, but I think that's ok? We need to visually validate it anyway, and I doubt that there will be many (if any) new minors in the first place.